src/caret_analyze/architecture/architecture_loaded.py

```diff
@@ -1608,6 +1637,15 @@ def __init__(
                     logger.info(msg)
                 paths += searched_paths
 
+            if skip_count:
+                logger.warn(
+                    f'contains callbacks whose construction_order are greater than'
+                    f' ({max_callback_construction_order_on_path_searching})'
+                    f': {node.node_name} ({skip_count=})'
+                    f' construction_order'
+                    f' ({construction_order["min"]} - {construction_order["max"]})'
```

Comment:
I suggest changing message as below to explain the situation clearly.

`{node.node_name} contains callbacks whose construction_order are greater than {max_callback_construction_order_on_path_searching}. {skip_count} paths are ignored as a result.`

I do not think min and max of construction_order are informative for user.

If you do not like this message, or this message is grammatically broken, please let me know.